internal/libs/clist/clist_test.go

```diff
@@ -320,7 +320,7 @@ FOR_LOOP:
 FOR_LOOP2:
 	for {
 		select {
-		case <-prev.prevWaitChan():
+		case <-prev.prevWaitCh:
```

Comment:
Ah, I was thinking we can just delete this whole for-loop since we don't expose this functionality at all.